### ISSUE TITLE
False alarm of 'disallowSpacesInsideAttributeBrackets' for multiline statements

### ISSUE THREAD

**Issue by TokugawaTakesi** [pugjs/pug-lint#181]
Documentation:

> Disallows space after opening attribute bracket and before closing.
> 
> ```pug
> //- Invalid
> input( type='text' name='name' value='value' )
> 
> //- Valid
> input(type='text' name='name' value='value')
> ```

If the rule [disallowSpacesInsideAttributeBrackets](https://github.com/pugjs/pug-lint/blob/master/docs/rules.md#disallowspacesinsideattributebrackets-true) aims on multiline  declarations too, it's not obvious. 

Live example:

```pug
mixin AttentionBox(propertiesObject)

  include AttentionBox-ComponentPropertiesSpecification

  -
    const {
      semanticType,
      CSS_Namespace,
      hasDefaultPrependedSVG_Icon,
      hasDismissButton
    } = MixinParametersValidator.getValidatedParameters(propertiesObject, AttentionBoxComponentPropertiesSpecification);

  div&attributes(attributes)(
    class= CSS_Namespace
    class= `${CSS_Namespace}${BEM_MODIFIER_SEPARATOR}${AttentionBox.SemanticTypesCSS_Modifiers[semanticType]}`
    role="alert"
  )

    if hasDefaultPrependedSVG_Icon
      svg(class=`${CSS_Namespace}${BEM_ELEMENT_SEPARATOR}PrependedSVG_Icon` viewBox="0 0 24 24")
        case semanticType
          when AttentionBox.SemanticTypes.importantInfo
            path(
              d="M13,13H11V7H13M13,17H11V15H13M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z"
            )
          when AttentionBox.SemanticTypes.secondaryInfo
            path(d="M13 14H11V9H13M13 18H11V16H13M1 21H23L12 2L1 21Z")
          when AttentionBox.SemanticTypes.error
            path(
              d="M13,13H11V7H13M13,17H11V15H13M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z"
            )
          when AttentionBox.SemanticTypes.warning
            path(d="M13 14H11V9H13M13 18H11V16H13M1 21H23L12 2L1 21Z")
```

![image](https://user-images.githubusercontent.com/41653501/108792489-bbd0df80-75c4-11eb-815e-42861ce86c38.png)

I tried that

```pug
            path(
              d="M13,13H11V7H13M13,17H11V15H13M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z"
)
```

is valid Pug syntax, but subjectively it makes code formatting ugly. I don't think that everyone wants this rule warn for multiline statements...
